langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -148,6 +247,15 @@ public isolated function filter(table<MapType> key<KeyType> t, @isolatedParam fu
 # The combining function takes the combined value so far and a member of the table,
 # and returns a new combined value.
 #
+# ```ballerina
+# table<record {| int id; int salary; |}> employees = table [
+#     {id: 1, salary: 1200},
+#     {id: 2, salary: 1100},
+#     {id: 3, salary: 800}
+# ];
+# employees.reduce(isolated function (int total, record {int id; int salary;} next) returns int => total + next.salary, 0) ⇒ 3100
```

Comment:
Why is this an inclusive record typedesc and that in the table an exclusive typedesc?